substrate/frame/bounties/src/lib.rs

```diff
@@ -1068,9 +1833,27 @@ impl<Balance: Zero> ChildBountyManager<Balance> for () {
 		Default::default()
 	}
 
+	fn children_value(_bounty_id: BountyIndex) -> Balance {
+		Zero::zero()
+	}
+
 	fn children_curator_fees(_bounty_id: BountyIndex) -> Balance {
 		Zero::zero()
 	}
 
 	fn bounty_removed(_bounty_id: BountyIndex) {}
 }
+
+/// TryConvert implementation to get the Source of the Bounties.
+pub struct BountySource<T, I = ()>(PhantomData<(T, I)>);
```

Comment:
the type name and the doc says nothing about the type of implementation but just repeats the config's type name